lib/models/package-info-cache/package-info.js

```diff
@@ -291,17 +291,22 @@ class PackageInfo {
   }
 
   dumpInvalidAddonPackages(addonPackageList) {
-    let invalidPackages = this.getInvalidPackages(addonPackageList);
+    if (!this.dumpedInvalidPackages) {
```

Comment:
2 nit picks:

* as this is private state, can we do `this._dumpedInvalidPackages`
* could we keep the guard isolated at the top of the method?

```
dumpInvalidAddonPackages(addonPackageList) {
  if (this._dumpedInvalidPackages) { return; }
  this._dumpedInvalidPackages = true;

  // rest of code
}
```

For two reasons:

* keeps the guard code close together
* meets our existing style guide

Reply:
Sure. I'll fix it. I'd forgotten that's how you like to do the guard code.